light_recipe/settings.py

```diff
@@ -36,6 +36,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+	
+	'recipes',
```

Comment:
this could be two different django applications. It is not necessary that these apps has the same areas of responsibilities